framework/Optimizers/SimulatedAnnealing.py

```diff
@@ -688,10 +697,11 @@ def _nextNeighbour(self, rlz,fraction=1,alpha = 0.94,cont = True):
         self.info['amp_'+var] = amp
         self.info['delta_'+var] = delta[i]
       self.info['fraction'] = fraction
+    """
+    TODO:
```

Comment:
"""

are not allowed in the code (it corrupts the doxygen)

+ you need to remove this if statement...when it is needed, we will add it.

Reply:
done